google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -546,4 +596,36 @@ public void setModelNeedsRefresh(boolean modelNeedsRefresh) {
       this.modelNeedsRefresh = modelNeedsRefresh;
     }
   }
+
+  /**
+   * Interface that must be implemented in order to be informed of
+   * {@link ProjectSelectionChangedEvent} events.
+   */
+  public interface ProjectSelectionListener {
```

Comment:
I like what you done here but I'm wondering if something should be refactored in the superclass `CustomizableComboBox` since its already setup for listening to selection changes?